### PR TITLE
Add CCN Main and Beta

### PR DESCRIPTION
**Pull request by hobbydev71**
Added the main and beta networks for CCN

### CODE REVIEW

File: _data/chains/eip155-971.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "rpc": [
+    "https://beta-rpc.mainnet.computecoin.com"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "ComputeCoin",
+    "symbol": "CCN",
+    "decimals": 18
+  },
+  "shortName": "CCN",
```
Comment (ligi) [inline _data/chains/eip155-971.json:14]:
```suggestion
  "shortName": "ccnbeta",
```

Comment (hobbydev71) [reply _data/chains/eip155-971.json]: I updated the shortname

**Review by ligi — CHANGES_REQUESTED** [review]
ShouldHaveNoMissingFields: missing field(s) [infoURL]

**Comment by hobbydev71** [comment]
> ShouldHaveNoMissingFields: missing field(s) [infoURL]

The infoURL is added.

**Comment by hobbydev71** [comment]
Hi @ligi ,
Could you check this PR to add new chain?

**Pull request closed without merging**